app/assets/stylesheets/resque_web/bootstrap_and_overrides.css.scss.erb

```diff
@@ -2,7 +2,6 @@
   =require twitter-bootstrap-static/bootstrap
   */
 
-@import "font-awesome-sprockets";
```

Comment:
So this doesn't break for everyone else wouldn't you want to do something like this?

```
/* app/assets/stylesheets/application.css.scss */
$rails_version: '#{asset-path("config/environment.rb")}';
@if ($rails_version < 6) {
  // Import styles only if the Rails version is less than 6
  @import "font-awesome-sprockets";
}
```

```
# config/application.rb or an initializer

# Set a custom environment variable to indicate the Rails version
Rails.application.config.rails_version = Rails.gem_version
```

I'm not entirely sure this works as written as it was generated by chatGPT but some author who knows scss syntax better than me could clean it up.